src/mrack/data/provisioning-config.yaml

```diff
@@ -0,0 +1,117 @@
+aws:
```

Comment:
I don't think we should have a default provisioning config with so many values specific to certain environment. Basically it should not contain anything which is not universal.

Reply:
I'm inclined to agree, does that mean we need "mostly" 1 image, 1 distro, 1 flavor etc?
Some projects have a default config file with lots of comment blocks, explaining every parameter with one (or none) default value enabled. As a user I would appreciate that.

Reply:
+1, commented examples could be good.

Reply:
@netoarmando Will do so